mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -165,3 +174,73 @@
 	{
 	}
 });
+
+::Reforged.HooksMod.hookTree("scripts/entity/tactical/actor", function(q) {
+	q.onActorKilled = @( __original ) function( _actor, _tile, _skill )
+	{
+		::Reforged.Config.XPOverride = true;
+		__original(_actor, _tile, _skill);
+		::Reforged.Config.XPOverride = false;
```

Comment:
This could become tricky when certain actors deal damage during this function which might cause additional onDeath functions for other actors to trigger.

The `flying_skull` from vanilla does this.
If they die and kill someone else with their damage then a nested onDeath is called. When that one finishes then the XPOverride is false but the first onDeath is still running and no longer applying our changed XP policy correctly.

On a side note. The Flying Skulls also cause other bugs due to their onDeath effects. For example you can cheese the Sunken Library by having your last brother die to a flying skull. Since the Skull was the last one to die (an enemy) you are attributed the win of this battle.